### PR TITLE
WIP: Switch to the Sphinx Book Theme

### PR DESCRIPTION
**Pull request by seisman**
Switch to the sphinx book theme but with minimal configurations.

### CODE REVIEW

**Comment by core-man** [comment]
Cool~
Some keyboards need to be revised.

https://seismo-learn.org/sitepreview/seismo-learn/seismology101/sphinx-book-theme/computer/fedora-setup/#id11

**Comment by core-man** [comment]
We cannot click the page to edit `.rst` directly on the website now.

**Comment by core-man** [comment]
We shall remove content in https://seismo-learn.org/sitepreview/seismo-learn/seismology101/sphinx-book-theme/best-practices/#

**Comment by core-man** [comment]
Command category is bad: https://seismo-learn.org/sitepreview/seismo-learn/seismology101/sphinx-book-theme/computer/commands/

File: source/conf.py [hunk 1]
```diff
@@ -80,9 +80,7 @@
 copybutton_remove_prompts = True
 
 # -- Options for HTML output -------------------------------------------------
-import sphinx_rtd_theme
-
-html_theme = "sphinx_rtd_theme"
+html_theme = "sphinx_book_theme"
```
Comment (core-man) [inline source/conf.py:83]: no need to import?

Comment (zhaozhiyuan1989) [reply source/conf.py:83]: 不用

**Comment by zhaozhiyuan1989** [comment]
这个主题对小图标支持是不是不太友好，:download: 之类的都不显示

**Comment by core-man** [comment]
> Command category is bad: https://seismo-learn.org/sitepreview/seismo-learn/seismology101/sphinx-book-theme/computer/commands/
> 
> We may try https://sphinx-panels.readthedocs.io/en/latest/.

Preview does not work.

**Comment by seisman** [comment]
> > Command category is bad: https://seismo-learn.org/sitepreview/seismo-learn/seismology101/sphinx-book-theme/computer/commands/
> > We may try https://sphinx-panels.readthedocs.io/en/latest/.
> 
> Preview does not work.

For unknown reasons, it still shows the old sites. Please wait for more minutes and check it again.

**Comment by core-man** [comment]
The theme does not work either.

**Comment by core-man** [comment]
Cool, I like this theme more and more. We can click the top button to hide the main content on the left.

Screenshot from 2021-02-03 13-50-00



Screenshot from 2021-02-03 13-50-07

**Comment by core-man** [comment]
One con: when we browse down from the top, the left content will be missing. I hope it can always be there.


Screenshot from 2021-02-03 13-52-26

**Comment by seisman** [comment]
> One con: when we browse down from the top, the left content will be missing. I hope it can always be there.

It works as you expect in their own site (https://sphinx-book-theme.readthedocs.io/en/latest/index.html). We may need to read the configurations.

**Comment by core-man** [comment]
Currently, we miss extern links:

Screenshot from 2021-02-03 14-09-18

Screenshot from 2021-02-03 14-10-20

See https://sphinx-book-theme.readthedocs.io/en/latest/configure.html#source-repository-buttons

**Comment by seisman** [comment]
> > One con: when we browse down from the top, the left content will be missing. I hope it can always be there.
> 
> It works as you expect in their own site (https://sphinx-book-theme.readthedocs.io/en/latest/index.html). We may need to read the configurations.

Compatibility issue, see https://github.com/executablebooks/sphinx-panels/issues/57

**Comment by seisman** [comment]
After migrating from sphinx-panels to sphinx-design, now the new theme looks great to me.

**Comment by core-man** [comment]
This page is not good: https://seismo-learn.org/sitepreview/seismo-learn/seismology101/sphinx-book-theme/computer/setup/

**Comment by seisman** [comment]
Closed by https://github.com/seismo-learn/seismology101/pull/507.

**Pull request closed without merging**